src/DynamoCoreWpf/UI/Themes/Modern/OutPorts.xaml

```diff
@@ -198,6 +198,30 @@
                 </Border.ToolTip>
 
             </Border>
+
+            <Border x:Name="PortBorderHighlight"
+                    Grid.Column="0"
+                    Grid.ColumnSpan="2"
+                    BorderBrush="Purple"
```

Comment:
I think there is a color value from @Jingyi-Wen 's design, it was mentioned in slack and embedded in https://github.com/DynamoDS/Dynamo/pull/15865. Below is for your reference:
`<SolidColorBrush x:Key="NodeTransientOverlayColor" Color="#D5BCF7" />`